Sources/TuistKit/Commands/TuistCommand.swift

```diff
@@ -14,28 +14,35 @@ public struct TuistCommand: AsyncParsableCommand {
         CommandConfiguration(
             commandName: "tuist",
             abstract: "Generate, build and test your Xcode projects.",
-            subcommands: [
-                BuildCommand.self,
-                CleanCommand.self,
-                DumpCommand.self,
-                EditCommand.self,
-                InstallCommand.self,
-                GenerateCommand.self,
-                GraphCommand.self,
-                InitCommand.self,
-                MigrationCommand.self,
-                PluginCommand.self,
-                RunCommand.self,
-                ScaffoldCommand.self,
-                TestCommand.self,
-                AuthCommand.self,
-                SessionCommand.self,
-                LogoutCommand.self,
-                CleanCommand.self,
-                ProjectCommand.self,
-                OrganizationCommand.self,
-                AnalyticsCommand.self,
-                CacheCommand.self,
+            subcommands: [],
+            groupedSubcommands: [
+                CommandGroup(name: "Start", subcommands: [
+                    InitCommand.self,
+                ]),
+                CommandGroup(name: "Develop", subcommands: [
+                    BuildCommand.self,
+                    CleanCommand.self,
+                    DumpCommand.self,
+                    EditCommand.self,
+                    InstallCommand.self,
+                    GenerateCommand.self,
+                    GraphCommand.self,
+                    MigrationCommand.self,
+                    PluginCommand.self,
+                    RunCommand.self,
+                    ScaffoldCommand.self,
+                    TestCommand.self,
+                    CleanCommand.self,
+                    CacheCommand.self,
+                ]),
+                CommandGroup(name: "Server", subcommands: [
```

Comment:
I wonder if it would be better to name this section as something like `Manage your Tuist account` instead of referring to the `Server`.

Reply:
Maybe `Account`? 🤷 :

Reply:
I like `Account`. I updated it.

Reply:
I still see `Server` :)